examples/case_studies/BART_quantile_regression.ipynb

```diff
@@ -0,0 +1,666 @@
+{
```

Comment:
<p>Shall we add <code>%config InlineBackend.figure_format = "retina"</code> to have higher quality on the images? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>